src/util/generic-tests.jsx

```diff
@@ -28,7 +28,8 @@ export function common(Component, {
 			};
 			const wrapper = shallow(<Component {...generateDefaultProps()} style={style}/>);
 			const rootWrapper = selector ? wrapper.find(selector).first() : wrapper.first();
-			assert.deepEqual(rootWrapper.prop('style'), style);
+			const rootStyle = rootWrapper.prop('style');
+			assert(_.every(style, (val, key) => val === rootStyle[key]), 'root style must contain passed styles');
```

Comment:
changed this so it just checks to make sure passed style is a subset of the final style object, rather than deep equal, since it should just get merged, rather than overwritten